lib/msal-angular/package.json

```diff
@@ -66,7 +66,8 @@
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
     "typescript": "~4.1.5",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.10.3",
+    "engine.io": ">=6.2.1"
```

Comment:
Don't think we should add a dependency to resolve this. I don't even see this dep in the package-lock so I'm not sure I understand where the alert came from. Maybe try re-running?